doc/doxygen/stylesheet.css

```diff
@@ -26,3 +26,11 @@ div.doxygen-generated-exception-message {
     padding: 5px;
     font-size: 95%;
 }
+
+div.image img[src="fe_enriched_1d.png"] {
+    width:500px;
```

Comment:
`@image html fe_enriched_1d.png "" width=500px` does not work either. So i guess we need to live with this solution until Doxygen gets more flexible.